router/h2/src/main/scala/com/twitter/finagle/buoyant/h2/H2FailureAccrualFactory.scala

```diff
@@ -0,0 +1,276 @@
+package com.twitter.finagle.buoyant.h2
+
+import com.twitter.finagle.Stack.{Params, Role}
+import com.twitter.finagle.buoyant.h2.service.{H2Classifier, H2ReqRep, H2ReqRepFrame}
+import com.twitter.finagle.client.Transporter
+import com.twitter.finagle.liveness.{FailureAccrualFactory, FailureAccrualPolicy}
+import com.twitter.finagle.service.ResponseClass
+import com.twitter.finagle.stats.StatsReceiver
+import com.twitter.finagle.{Status => FStatus, param => fParam, _}
+import com.twitter.logging.Level
+import com.twitter.util._
+import io.buoyant.router.context.h2.H2ClassifierCtx
+
+/* Modified from https://github.com/twitter/finagle/blob/develop/finagle-core/src/main/scala/com/twitter/finagle/liveness/FailureAccrualFactory.scala */
```

Comment:
This file is identical to FailureAccrualFactory except for the addition of `classifyResponse` and `classifyStream` and a modified `apply` method.